projectsmember/Pages/CooperationRequest/Profiles/HamidForoughnia.html

```diff
@@ -0,0 +1,163 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> H.Foroughnia's Online Resume</title>
+    <link href="../../../Content/Bootstrap/bootstrap.css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+
+    <style>
+        .material-symbols-outlined {
+            font-variation-settings:
+                    'FILL' 0,
+                    'wght' 400,
+                    'GRAD' 0,
+                    'opsz' 48
+        }
+    </style>
+
+</head>
+<body  dir="rtl">
+<div class="container border">
+    <div class="row">
+        <h1 class="mt-4">حمید فروغ نیا</h1>
+        <a href="https://dotnet.microsoft.com/en-us/languages/csharp" rel="nofollow">
+            <p>برنامه نویس c#</p>
+        </a>
+    </div>
+    <div class="row">
+        <div class="col">
+            <span class="material-symbols-outlined fs-6 fw-lighter">
+                <p style="display: inline-block;">
+                    استان قزوین، قزوین، منطقه ۳
+                </p>
+                location_on
+            </span>
+        </div>
+        <div class="col">
+            <span class="material-symbols-outlined fs-6 fw-lighter">
+                <img src="Mail.png" alt="">
```

Comment:
You need to also add this image - I know I mentioned earlier to only add your HTML file, but this image is also necessary for your webpage